packages/support/lib/fs.js

```diff
@@ -36,21 +37,48 @@ const findRootCached = _.memoize(pkgDir.sync);
 const fs = {
   /**
    * Resolves `true` if `path` is _readable_, which differs from Node.js' default behavior of "can we see it?"
-   * @param {import('fs').PathLike} path
+   *
+   * On Windows, ACLs are not supported, so we only check for file existence.
+   * @param {PathLike} path
    * @returns {Promise<boolean>}
    */
   async hasAccess(path) {
     try {
-      await fsPromises.access(path, constants.R_OK);
-    } catch (err) {
+      if (isWindows()) {
+        await fsPromises.stat(path);
+      } else {
+        await fsPromises.access(path, constants.R_OK);
+      }
+    } catch {
+      return false;
+    }
+    return true;
+  },
+
+  /**
+   * Resolves `true` if `path` is executable; `false` otherwise.
+   *
+   * On Windows, ACLs are not supported; resolves `true` if `path` exists.
+   *
+   * This function will _not_ reject if the file does not exist.
+   * @param {PathLike} path
+   * @returns {Promise<boolean>}
+   */
+  async isExecutable(path) {
+    try {
+      if (isWindows()) {
+        return await fs.hasAccess(path);
```

Comment:
so on Windows it is just enough to verify the file exists to assume it is executable?

Reply:
I don't know of any way to check this with node. it's possible there's some 3p module that has better windows support, but I don't know of one

Reply:
I think it would be simpler to explicitly mention in the docs that this method only makes sense for *nix and on Windows it would always return the same value.

Reply:
I'm confused; it doesn't always return the same value. It just defers to `hasAccess()`.  I'll change my wording anyway